compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -1565,7 +1566,8 @@ void startInvocationNode(Set<Whitespace> ws) {
         invocationWsStack.push(ws);
     }
 
-    void createInvocationNode(DiagnosticPos pos, Set<Whitespace> ws, String invocation, boolean argsAvailable) {
+    void createInvocationNode(DiagnosticPos pos, Set<Whitespace> ws, String invocation, boolean argsAvailable,
+                              DiagnosticPos identifierPos) {
```

Comment:
@hasithaa This is going to 1.0.x branch right? Is this API change ok?

Reply:
Since this is not a public method, it is patch compatible.